src/test/java/com/wikia/webdriver/testcases/mcfootertests/EnAnonMixedContentFooterTests.java

```diff
@@ -115,15 +117,16 @@ public void countNoOfArticlesInExploreCard() {
     Assertion.assertEquals(mcFooter.countArticlesInExploreCard(), 3);
   }
 
-  @Test
+  @Test(groups = {"test_aga"})
   public void userIsTakenToDiscussionsAfterClickOnViewAll() {
-    DiscussionsPage discussions = new MixedContentFooter()
+//    DiscussionsPage discussions =
+        new MixedContentFooter()
         .openWikiMainPage()
-        .scrollToMCFooter()
-        .getDiscussionsCard()
-        .clickOnViewAllLinkInDiscussions();
-
-    Assertion.assertTrue(discussions.isDiscussions());
+        .scrollToMCFooter();
+//        .getDiscussionsCard()
+//        .clickOnViewAllLinkInDiscussions();
+//
+//    Assertion.assertTrue(discussions.isDiscussions());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)